### PR TITLE
Duplicate statement in BaseBackupFailureTest.java

### PR DESCRIPTION
**Pull request by kilida**
https://issues.jboss.org/browse/ISPN-6440

### CODE REVIEW

**Comment by slaskawi** [comment]
Hey @kilida! Could you please create a JIRA for it: https://issues.jboss.org/browse/ispn

**Comment by vjuranek** [comment]
@slaskawi Do we really need JIRAs even for such simple and obvious changes?

File: core/src/test/java/org/infinispan/xsite/backupfailure/BaseBackupFailureTest.java [hunk 1]
```diff
@@ -59,7 +59,6 @@ public void reset() {
          prepareFailed = false;
          putFailed = false;
          removeFailed = false;
-         removeFailed = false;
```
Comment (danberindei) [inline core/src/test/java/org/infinispan/xsite/backupfailure/BaseBackupFailureTest.java]: I think this should be `replaceFailed = false` instead.

**Comment by slaskawi** [comment]
Pulling and squashing...

**Comment by slaskawi** [comment]
Integrated thanks!

**Pull request closed without merging**